pkg/operator/ceph/controllers/nodedrain/reconcile.go

```diff
@@ -0,0 +1,119 @@
+package nodedrain
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/rook/rook/pkg/operator/ceph/controllers/controllerconfig"
+
+	"github.com/coreos/pkg/capnslog"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var log = capnslog.NewPackageLogger("github.com/rook/rook", "nodedrain-controller")
```

Comment:
s/log/logger/

So we stay consistent with the rest of the codebase.

Reply:
+1